.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: maven
+    directory: /
+    schedule:
+      interval: weekly
```

Comment:
![image](https://user-images.githubusercontent.com/102726227/194803198-b2b4cb60-c890-4891-923a-10dc98190374.png)
New line is required at the end of file here. It can lead to unexpected behavior when using other tools on the file. 
for ref: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log